src/index.ts

```diff
@@ -49,52 +49,65 @@ class ConnDB {
 
   //#region INTERNAL
 
-  private _init(modernPath: string, legacyPath: string) {
-    const modernExists = fs.existsSync(modernPath);
-    const legacyExists = fs.existsSync(legacyPath);
-
-    if (!modernExists && !legacyExists) {
-      this._stateFile!.set({}, () => {});
-      this._loadedResolve(true);
-      debug(
-        'Created new conn.json because there was no existing ' +
-          'conn.json nor gossip.json',
-      );
-      return;
+  private _fileExists(path: string, cb: (exists: boolean) => void) {
+    if (typeof localStorage === "undefined" || localStorage === null)
+      fs.exists(path, cb)
```

Comment:
`exists` is deprecated in Node.js while `existsSync` is not, so it would be good to preserve `existsSync` here. Who knows if in new version of Node.js `exists` stops working entirely.

I can do this change after merging, no worries.

Reply:
Thanks